### PR TITLE
Closure cleanup part 3

### PR DESCRIPTION
**Pull request by juj**

### CODE REVIEW

File: src/jsifier.js [hunk 1]
```diff
@@ -58,6 +58,11 @@ function demangleCSymbolName(f) {
   return f[0] == '_' ? f.substr(1) : '$' + f;
 }
 
+function isJsLibraryConfigIdentifier(ident) {
```
Comment (kripken) [inline src/jsifier.js]: This looks like code from an unmerged PR? Or complementary to it.

As I mentioned in that PR, I think this can also be used in `library_webgl.js`.

File: src/support.js [hunk 2]
```diff
@@ -186,30 +186,30 @@ function loadDynamicLibrary(lib, flags) {
   LDSO.loadedLibNames[lib] = handle;
   LDSO.loadedLibs[handle] = dso;
 
-  // libData <- lib
-  function loadLibData() {
+  // libData <- libraryData
+  function loadLibData(libraryData) {
```
Comment (kripken) [inline src/support.js]: maybe `libFile` as it's the filename?

File: src/library_syscall.js [hunk 3]
```diff
@@ -1039,7 +1039,7 @@ var SyscallsLibrary = {
   __syscall203__sig: 'ii',
   __syscall203__nothrow: true,
   __syscall203__proxy: false,
-  __syscall203: '__sysicall214',     // setreuid32
```
Comment (sbc100) [inline src/library_syscall.js]: Is this just a typo that has ways been there?     What happens if you try to call this function with such a typo extant?

File: src/support.js (hunk 2 above)
Comment (sbc100) [reply src/support.js]: or even better just `filename`?

File: src/library.js [hunk 4]
```diff
@@ -4836,6 +4836,7 @@ LibraryManager.library = {
   emscripten_asm_const_double_sync_on_main_thread: function() {},
   emscripten_asm_const_async_on_main_thread: function() {},
 
+#if !WASM_BACKEND
   // ======== compiled code from system/lib/compiler-rt , see readme therein
```
Comment (sbc100) [inline src/library.js]: Unrelated?

File: src/library_syscall.js (hunk 3 above)
Comment (juj) [reply src/library_syscall.js]: Yes, this was in the tree - you'll probably get an undefined function error?

File: src/library.js (hunk 4 above)
Comment (juj) [reply src/library.js]: No, this is needed or Closure will complain that the generated functions below have unnecessary arithmetic operations.

File: src/library_trace.js [hunk 5]
```diff
@@ -119,7 +119,7 @@ var LibraryTracing = {
     googleWTFEnterScope: function(name) {
       var scopeEvent = EmscriptenTrace.googleWTFData['cachedScopes'][name];
       if (!scopeEvent) {
-        scopeEvent = window.wtf.trace.events.createScope(name);
+        scopeEvent = window['wtf'].trace.events.createScope(name);
```
Comment (jasonsanjose) [inline src/library_trace.js]: FWIW I trying shimming this into our project (currently using version 1.38.47) and closure was having problems with variable name mangling:

* Usage of `window['wtf'].trace.events.createScope` was mangled
* Usage of `EmscriptenTrace.googleWTFData['cachedScopes']` wasn't mangled, but the declaration of `cachedScopes` was mangled.

Comment (juj) [reply src/library_trace.js]: This probably needs something like `window['wtf']['trace']['events']`, or a Closure externs file. I have not used Google's WTF with closure (probably nobody else has either since this has not come up), probably needs fixing in several places.
